nuget/runners/nunit.console-runner.nuspec

```
@@ -26,10 +26,24 @@
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
-    <file src="bin/net20/nunit-agent.exe" target="tools" />
-    <file src="bin/net20/nunit-agent.exe.config" target="tools" />
-    <file src="bin/net20/nunit-agent-x86.exe" target="tools" />
-    <file src="bin/net20/nunit-agent-x86.exe.config" target="tools" />
+    <file src="bin/agents/net20/nunit-agent.exe" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit-agent.exe.config" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit-agent-x86.exe" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit-agent-x86.exe.config" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit.engine.api.dll" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit.engine.api.xml" target="tools/agents/net20" />
+    <file src="bin/agents/net20/nunit.engine.core.dll" target="tools/agents/net20" />
+    <file src="bin/agents/net20/Mono.Cecil.dll" target="tools/agents/net20" />
+    <file src="..\..\nuget\runners\nunit.console.nuget.addins" target="tools/agents/net20"/>
+    <file src="bin/agents/net40/nunit-agent.exe" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit-agent.exe.config" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit-agent-x86.exe" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit-agent-x86.exe.config" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit.engine.api.dll" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit.engine.api.xml" target="tools/agents/net40" />
+    <file src="bin/agents/net40/nunit.engine.core.dll" target="tools/agents/net40" />
+    <file src="bin/agents/net40/Mono.Cecil.dll" target="tools/agents/net40" />
+    <file src="..\..\nuget\runners\nunit.console.nuget.addins" target="tools/agents/net40"/>
```

I know this was just copy-paste, but while we're here, please can we switch them to cross-platform friendly path separators? 😁

Good job remembering to include the addins files though...not sure I would have. 

```suggestion
    <file src="../../nuget/runners/nunit.console.nuget.addins" target="tools/agents/net20"/>
    <file src="bin/agents/net40/nunit-agent.exe" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit-agent.exe.config" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit-agent-x86.exe" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit-agent-x86.exe.config" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit.engine.api.dll" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit.engine.api.xml" target="tools/agents/net40" />
    <file src="bin/agents/net40/nunit.engine.core.dll" target="tools/agents/net40" />
    <file src="bin/agents/net40/Mono.Cecil.dll" target="tools/agents/net40" />
    <file src="../../nuget/runners/nunit.console.nuget.addins" target="tools/agents/net40"/>
```

---

Ack - on second thoughts, the addin's file is going to break isn't it, cos the relative paths inside it will be starting from a directory a level deeper. 

In the long run, we need to work on that solution to having agents not load their own extensions. In the short term - how about creating a separate `nunit.agents.nuget.addins` file?

---

Right on both counts!

---

Oh! Were you referring to the discussion we had about having the extensions located by the engine, which would leave info in the `TestPackage` for the agent to find?

---

OK, I added a file into each of the agent directories containing only `../../` in order to forward to the addins file in the root directory. I'm going to build and test that.

---

This works as expected. By using `../../` in each agent, the actual content of the the main addins file doesn't have to be replicated.